manual.asc

```diff
@@ -845,6 +845,22 @@ Double the scrolling distance when using the eReader hardware buttons
 
 
 == Advanced Features
+[[mathjax]]
+=== MathJax Support
+Mathjax is a modern typesetting library for math and chemistry.  AnkiDroid support Mathjax cards out of the box as of AnkiDroid 2.9.  AnkiDroid follows the Mathjax configuration that Anki Desktop and AnkiMobile use.  Mathjax is configured to expect TeX formatting, with `\(` and `\)` delimiting inline equations and `\[` and `\]` for display equations.
+
+An example (but nonsensical) card could have the front of:
```

Comment:
I think this example is unnecessarily confusing. I propose copying the same example and most of the same wording from the Anki Desktop manual.